firebase.json

```diff
@@ -6,46 +6,56 @@
       "**/.*",
       "**/node_modules/**"
     ],
-    "headers": [ {
-      "source" : "**",
-      "headers" : [ {
-        "key" : "Access-Control-Allow-Origin",
-        "value" : "*"
-      },
-      {
-        "key" : "Access-Control-Allow-Methods",
-        "value" : "POST, GET, OPTIONS, HEAD"
-      }      
-     ]
-    },
-   {
-      "source" : "**/**",
-      "headers" : [ {
-        "key" : "Access-Control-Allow-Origin",
-        "value" : "*"
-      },
+    "headers": [
       {
-        "key" : "Access-Control-Allow-Methods",
-        "value" : "POST, GET, OPTIONS, HEAD"
-      }      
-     ]
-    },
-    {
-      "source" : "cert/*.json",
-      "headers" : [ {
-        "key" : "Access-Control-Allow-Origin",
-        "value" : "*"
+        "source": "**",
+        "headers": [
+          {
+            "key": "Access-Control-Allow-Origin",
+            "value": "*"
+          },
+          {
+            "key": "Access-Control-Allow-Methods",
+            "value": "POST, GET, OPTIONS, HEAD"
+          }
+        ]
       },
       {
-        "key" : "Access-Control-Allow-Methods",
-        "value" : "POST, GET, OPTIONS, HEAD"
+        "source": "**/**",
+        "headers": [
+          {
+            "key": "Access-Control-Allow-Origin",
+            "value": "*"
+          },
+          {
+            "key": "Access-Control-Allow-Methods",
+            "value": "POST, GET, OPTIONS, HEAD"
+          }
+        ]
       },
       {
-        "key" : "Access-Control-Allow-Headers",
-        "value" : "*"
-      }          
-     ]
-    }        
+        "source": "cert/*.json",
+        "headers": [
+          {
+            "key": "Access-Control-Allow-Origin",
+            "value": "*"
+          },
+          {
+            "key": "Access-Control-Allow-Methods",
+            "value": "POST, GET, OPTIONS, HEAD"
+          },
+          {
+            "key": "Access-Control-Allow-Headers",
+            "value": "*"
+          }
+        ]
+      }
+    ]
```

Comment:
I think you need to add :
"rewrites": [ {
      "source": "/metadata/**",
      "function": "metadata"
    },

Reply:
done